packages/nextjs/app/builders/0x2d90C8bE0Df1BA58a66282bc4Ed03b330eBD7911/page.tsx

```diff
@@ -0,0 +1,149 @@
+"use client";
```

Comment:
We don't need this restriction since your page doesn't use any client-only mechanism.
Please remove this line so we can improve the app's performance by pre-rendering the page during the compile process.

Reply:
Please, remove this restriction.